.circleci/config.yml

```diff
@@ -1,37 +1,25 @@
 ---
 version: 2.1
 orbs:
-  ruby-rails: sul-dlss/ruby-rails@4.4.0
-  browser-tools: circleci/browser-tools@1.5.0
-  node: circleci/node@7.0.0
-  ruby: circleci/ruby@2.3.1
-  cypress: cypress-io/cypress@2.2.0
+  ruby-rails: sul-dlss/ruby-rails@4.5.0
+  browser-tools: circleci/browser-tools@1.5.2
+  ruby: circleci/ruby@2.4.0
+  cypress: cypress-io/cypress@3.4.3
 jobs:
   cypress:
     executor: ruby-rails/ruby-postgres
     steps:
       - checkout
       - ruby/install-deps
       - cypress/install:
-          yarn: true
-          post-install:
-            - run: sudo apt-get update
-            - run: sudo apt-get install --fix-missing libgtk2.0-0 libgtk-3-0 libgbm-dev
-                libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6
-                xauth xvfb
-      - browser-tools/install-firefox:
-          version: 115.9.0esr
-      - node/install-packages:
-          pkg-manager: yarn
-      - run:
-          name: Install javascript dependencies and build assets
-          command: bin/rake test:prepare
-      - run:
-          command: dockerize -wait tcp://localhost:5432 -timeout 1m
-          name: Wait for DB
-      - run:
-          command: bundle exec bin/cypress run
-          name: Test
+          package-manager: yarn
```

Comment:
We were like 20 versions behind in Firefox. Bumping to the latest Firefox and Cypress versions led me down a bit of an orb rabbithole, but what we have now is terser, uses the orbs to a greater extent, and is hopefully more readable.